.github/workflows/ci.yaml

```diff
@@ -41,22 +41,34 @@ jobs:
           - "3.11"
           - "3.12"
         openeye: ["no"]
+        gufe-version: ['a9b5982471eb3644ca8c4423c7001e6309a3f544']
         include:
           - os: "macos-latest"
             python-version: "3.12"
             pydantic-version: ">1"
+            gufe-version: 'a9b5982471eb3644ca8c4423c7001e6309a3f544'
           - os: "ubuntu-latest"
             python-version: "3.11"
             pydantic-version: "<2"
+            gufe-version: 'a9b5982471eb3644ca8c4423c7001e6309a3f544'
           - os: "ubuntu-latest"
             python-version: "3.11"
             pydantic-version: ">1"
             openeye: "yes"
+            gufe-version: 'a9b5982471eb3644ca8c4423c7001e6309a3f544'
           - os: "macos-latest"
             python-version: "3.12"
             pydantic-version: ">1"
             omff-version: ">0.13"
             openeye: ["no"]
+            gufe-version: 'a9b5982471eb3644ca8c4423c7001e6309a3f544'
+
+          # temporary test against gufe main while we're pinned to a9b5
+          - os: "ubuntu-latest"
```

Comment:
@IAlibay do you have an opinion about what build to test here?

Reply:
I'm not sure I fully understand what you mean - this seems fine to me!

Reply:
just what combination of os, openeye, etc. would be most useful, since we're just choosing one here.

Reply:
This is good, it covers the widest range of tests :)